src/lv_font/lv_font_fmt_txt.c

```diff
@@ -322,8 +317,8 @@ static int32_t kern_pair_8_compare(const void * ref, const void * element)
     const uint8_t * element8_p = element;
 
     /*If the MSB is different it will matter. If not return the diff. of the LSB*/
-    if(ref8_p[0] != element8_p[0]) return (int32_t)ref8_p[0] - element8_p[0];
```

Comment:
It seemed correct. The values are always big-endian. E.g.
```c
/*Pair left and right glyphs for kerning*/
static const uint16_t kern_pair_glyph_ids[] =
{
    1, 34,
    1, 53,
    1, 58,
    18, 18,
    34, 1,
    34, 53,
    34, 55,
    34, 56,
    34, 58,
    34, 87,
    34, 88,
    34, 90,
    34, 96,
```

Reply:
Yes, you are right. I misunderstand the spec and comment.